python/vTestInterface.py

```diff
@@ -140,8 +140,13 @@ def generateTestInfo(test):
 
 def getTestDataVCAST(enviroPath):
 
-    api = UnitTestApi(enviroPath)
-
+    # dataAPI throws if there is a tool/enviro missmatch
+    try:
+        api = UnitTestApi(enviroPath)
+    except Exception as err:
+        print (err)
+        raise Exception("INVALID_ENVIRO")
```

Comment:
Probably, the best way to do this is to make an `InvalidEnviro` exception and throw that.

At least, we should make `"INVALID_ENVIRO"` and `"USAGE_ERROR"` be string constants and re-use those rather than duplicating the strings.

Reply:
Done